python/GafferCortexUI/OpDialogue.py

```diff
@@ -180,7 +180,7 @@ def __init__(
 
 			with GafferUI.Collapsible( "Details", collapsed = True ) as self.__messageCollapsible :
 
-				self.__messageWidget = GafferUI.MessageWidget()
+				self.__messageWidget = GafferUI.MessageWidget( toolbars = True )
```

Comment:
What's the reason for doing this piecemeal at each call site, rather than changing the default? The default in 0.57 was for similar (but not as good) functionality to be on by default. Looking at the other uses in Gaffer, we'd definitely want the toolbars in ErrorDialogue and probably NotificationMessageHandler, but maybe not in ExpressionUI and OSLCodeUI?

Reply:
I think we just thought at the time, we'd be more conservative opt-in to the mode complex UI, rather than opting out. Primarily to avoid showing severity controls when they're not needed (the old one didn't show the extra UI if there was only one severity).

Reply:
OK, then let's add an opt-in for ErrorDialogue before we merge. Could you rebase to fix the merge conflicts at the same time please?

Reply:
Added [here](https://github.com/GafferHQ/gaffer/pull/3934/commits/19826bf8504ca3d2c6fc9b2c0fa7b695ac2442ea)

Reply:
Thanks!